### PR TITLE
Rename the User menu to Account menu.

### PR DESCRIPTION
**Pull request by jenlampton**
PR for https://github.com/backdrop/backdrop-issues/issues/303

### CODE REVIEW

File: core/modules/menu/menu.install [hunk 1]
```diff
@@ -48,7 +48,7 @@ function menu_install() {
   $t = get_t();
   $descriptions = array(
     'navigation' => $t('The <em>Navigation</em> menu contains links intended for site visitors. Links are added to the <em>Navigation</em> menu automatically by some modules.'),
-    'user-menu' => $t("The <em>User</em> menu contains links related to the user's account, as well as the 'Log out' link."),
+    'user-menu' => $t("The <em>Account</em> menu contains links related to eah user's account, as well as links to 'Log in' and 'Log out'."),
```
Comment (quicksketch) [inline core/modules/menu/menu.install]: > "related to eah user's account"

Introduced a typo here.

**Comment by jenlampton** [comment]
Good catch!  Fixed.

File: core/includes/menu.inc [hunk 2]
```diff
@@ -1739,7 +1739,8 @@ function menu_list_system_menus() {
   return array(
     'main-menu' => 'Main menu',
     'management' => 'Management',
-    'user-menu' => 'User menu',
+    'user-menu' => 'Account menu',
+    'main-menu' => 'Main menu',
```
Comment (rocketnova) [inline core/includes/menu.inc]: It seems wrong that this is adding another entry for 'main-menu'.

Comment (jenlampton) [inline core/includes/menu.inc]: good catch, I think it was supposed to be re-ordering them, I bet the merge cleanup didn't go smoothly.

**Comment by jenlampton** [comment]
Addressed comments by @rocketnova, rebased and re-pushed branch. testing now.

**Pull request merged**